app/scripts/react-directives/common.jsx

```
@@ -103,11 +103,18 @@
   );
 };
 
-export const Button = ({ label, type, onClick, disabled, additionalStyles, icon, title }) => {
+export const Button = ({ label, type, onClick, disabled, additionalStyles, icon, title, form }) => {
   const classes =
     'btn btn-' + (type ? type : 'default') + (additionalStyles ? ' ' + additionalStyles : '');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)